{"name": "test_put_invalid_data_type", "status": "passed", "description": "\n        Проверка статуса ответа метода PUT при отправлении данных в невалидном формате.\n        Ожидаемый результат 400\n        ", "attachments": [{"name": "log", "source": "a3225386-c9e6-48cc-82d3-3007fbbe3bcc-attachment.txt", "type": "text/plain"}], "start": 1591441674893, "stop": 1591441674901, "uuid": "81e403ee-8ec4-4055-92db-0d8391506011", "historyId": "fa299cd90ee59a51115266ccd03ce966", "testCaseId": "2f07d9743a249563e36e87a0437a47f8", "fullName": "tests.test_api_put.TestPut#test_put_invalid_data_type", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api_put"}, {"name": "subSuite", "value": "TestPut"}, {"name": "host", "value": "MacBook-Air-Mac.local"}, {"name": "thread", "value": "5340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api_put"}]}